docs/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 ## Next
+* Add light level filters to the loadout search.
+  * `light:` filter finds loadouts whose light level matches the provided range, for loadouts where all weapon and armor slots have items assigned.
+  * `armorlight:` filter finds loadouts where the armor light level matches the provided range, for loadouts where all armor slots have items assigned.
+  * `weaponlight:` filter finds loadouts whose weapon light level matches the provided range, for loadouts where all weapon slots have items assigned.
```

Comment:
I'm not as sure about these two - specifically, I'm not sure of their actual utility since an incomplete loadout doesn't really have a light level. We also don't show this in the UI today, so you can't really double-check that the filter is working properly.

Reply:
Yeah, I honestly wasn't sure about them either. 

There is utility in being able to search loadouts that contain only armor or weapons, but at the same time the utility falls off sharply as items get leveled to near the power cap. And yeah, as you pointed out - they aren't complete loadouts so they don't really have an actual light level (even though we can calculate something like one).

They're easy enough to re-implement, so I don't mind removing them.